### ISSUE TITLE
"id" prop should be renamed to "input-id"

### ISSUE THREAD

**Issue by h6ah4i** [phoenixwong/vue2-timepicker#18]
Hi. I noticed that something weird bug with this library.

When I using the library through pre-built version (`dist/vue2-timepicker.js`), the following error message is displayed on console;

<img src="https://cloud.githubusercontent.com/assets/2552365/24079503/dbe1c736-0ccc-11e7-9acf-60ef28e2410a.png" width="500" />

**Note that, this error is only present when I use pre-built library, it does not reproduce when using `import`(or 'require').**

---

I checked the `dist/vue2-timepicker.js` and I found out that the `id` prop definition is removed from the prebuilt library.

#### dist/vue2-timepicker.js
```javascript
    var VueTimepicker = Vue.extend({
      name: 'VueTimepicker',

      props: {
        value: {type: Object},
        hideClearButton: {type: Boolean},
        format: {type: String},
        minuteInterval: {type: Number},
        secondInterval: {type: Number}   // !!! Oh... there should be the id prop on the nest line :(
      },

      data () {
        return {
          hours: [],
         ...
```

I think we should not use `id` prop for custom component, because it is reserved by Vue. Also, the current specification is problematic because we cannot specify `id` prop to the VueTimepicker component itself.

IMO, renaming the `id` prop to `input-id` is the most relevant solution in this case. (sadly, this is a breaking API change though...)

**Comment by phoenixwong** [comment]
Sorry for the long wait. It's due to the wrongly bundled (dist) file in v0.1.3 and v0.1.4. Vue does allow using `id` in `props`. Latest release [v0.2.0](https://github.com/phoenixwong/vue2-timepicker/releases/tag/v0.2.0) fixes this issue now (#66).

Which means we can continue using:

```
<vue-timepicker id="yourOwnId"></vue-timepicker>

<vue-timepicker :id="idVar"></vue-timepicker>
```

**Comment by phoenixwong** [comment]
Duplicate of #10
